### ISSUE TITLE
Playing waveforms on multiple bias lines simultaneously

### ISSUE THREAD

**Issue by jlashner** [slaclab/pysmurf#571]
## Describe the problem
We're unable to play waveforms on multiple bias lines simultaneously, making it time consuming to run bias line tests on all bias groups. 

## Describe the solution you'd like
It would be great to have a way of playing a waveform on multiple bias lines simultaneously. Having a way to play different waveforms on different bias lines would be ideal, but a shared waveform will still be a huge improvement.

## Additional details
Here's an issue in the smurf-issues repository about trying to play a sine wave on multiple lines: https://github.com/simonsobs/smurf-issues/issues/14. I believe the behavior where the bias does not return to zero has already been logged here: https://github.com/slaclab/pysmurf/issues/425

**Comment by ruck314** [comment]
Reviewed against pysmurf `main` at commit `5f0b34eb`.

**Status: not implemented and not implementable in pysmurf alone.** Recommend closing as `wontfix` (the existing label).

### What main has today
The only TES-bias waveform entry point is `play_tes_bipolar_waveform(bias_group, waveform, ...)` in [`python/pysmurf/client/util/smurf_util.py` lines 4259-4310](https://github.com/slaclab/pysmurf/blob/main/python/pysmurf/client/util/smurf_util.py#L4259-L4310). It targets exactly one bias group:

```python
self.set_dac_axil_addr(0, dac_positive)   # LUT 0 -> + DAC of this group
self.set_dac_axil_addr(1, dac_negative)   # LUT 1 -> - DAC of this group
self.set_rtm_arb_waveform_enable(3)       # both LUTs on
...
self.set_rtm_arb_waveform_lut_table(0,  waveform)
self.set_rtm_arb_waveform_lut_table(1, -waveform)
```

### Why this can't be fixed in pysmurf
The RTM arbitrary waveform engine in firmware exposes only **two LUT slots**, and the engine's enable register has only four states:

- [`smurf_command.py:3697-3715`](https://github.com/slaclab/pysmurf/blob/main/python/pysmurf/client/command/smurf_command.py#L3697-L3715) — `DacAxilAddr[reg]` asserts `reg in range(2)` (LUT 0 or LUT 1, no third slot).
- [`smurf_command.py:3777-3794`](https://github.com/slaclab/pysmurf/blob/main/python/pysmurf/client/command/smurf_command.py#L3777-L3794) — `EnableCh ∈ {0, 1, 2, 3}` (`0x3` is "both LUT 0 and LUT 1"; there is no bit for a third or fourth channel).

A single bipolar bias group consumes **both** LUTs (positive DAC ← LUT 0, inverted negative DAC ← LUT 1), so no LUT capacity remains for any additional group. The LUT→DAC mapping is also strictly 1:1 (one `DacAxilAddr[reg]` register per LUT slot), so even the consolation request — "a shared waveform on multiple bias lines" — has no firmware fan-out path either.

### Open PRs
Searched open PRs for waveform / bias-line / `#571`: no PR adds multi-group simultaneous waveform support. The only adjacent open PR is #895 ("Enable waveform output last (#678)"), which fixes a sequencing bug in the existing single-group function and does not change capability.

### What would actually unblock this
This requires firmware work in the RTM arbitrary waveform engine — either additional LUT tables (one bipolar pair per supported simultaneous bias group) or an explicit fan-out mechanism that lets a single LUT pair drive multiple DAC pairs. Until that lands, pysmurf has nothing to expose. Suggest closing as `wontfix` and reopening (or filing a new issue) against the firmware repo if/when the engine is extended.
